optimum/onnx/graph_transformations.py

```diff
@@ -92,7 +59,7 @@ def remove_duplicate_weights(model: ModelProto, inplace: bool = False) -> ModelP
     return model
 
 
-def replace_atenops_to_gather(model: ModelProto):
+def replace_atenops_to_gather(model: ModelProto) -> ModelProto:
```

Comment:
Does this replace aten ops related to Gather to a working version?
If so I would make it more explicit in the name.

Reply:
No idea, I did not code it.